dotCMS/src/main/java/com/dotcms/workflow/form/WorkflowActionForm.java

```diff
@@ -75,8 +76,8 @@ public boolean isRequiresCheckout() {
         return requiresCheckout;
     }
 
-    public RequiresCheckoutOption getRequiresCheckoutOption() {
-        return requiresCheckoutOption;
+    public Set<WorkflowStatus> getShowOn() {
+        return showOn;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "showOn". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)

Reply:
DONE